src/TraceEvent/Ctf/CtfTracing.Tests/CtfTracing.Tests.csproj

```diff
@@ -36,19 +36,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
   </ItemGroup>
-  <Choose>
```

Comment:
> Removing support for 2015 was an intentional design change.

VS 2017 is really quite new (it has been out only since march).   Dropping compatibility for any previous version of VS should not be done lightly.     Your second point about about .NET Core is interesting and may be a good enough reason, but what is also true is that perfView itself (as opposed building a .NET Core compatible TraceEvent), does not require .NET Core.   Making .NET Core projects require 2017 is OK.   

In general whenever there is a non-trivial take away, a more formal process is needed.   It is one thing to add support for 2017, it is another to remove support for 2015.    To do the later, we need to warn people, and probably provide useful guidance on workarounds (e.g. a label of the last place in the code base  where VS2015 worked).  

Now if it was HARD to make 2017 work without dropping support for 2015, then we probably would warn people and make the switch, but from what I can tell, it is NOT that hard (The main issue I ran into when I tried it was the EDITBIN, which you point out we can simply remove).    

I realize that unless we set the language version to #6, reasonably quickly C#-7 constructs will be introduced into the code base that effectively breaks VS2015 users (although maybe not, I don't know if we would end up shipping VS2015 updates which bring C# 7 to VS2015).   But still, I feel there is value in keeping build system and other 'non-code' changes compatible (which allows people to fork and selectively cherry-pick more effectively).   

So how to proceed:    First I would like to understand if we can support BOTH without too much trouble.  If this takes less than a couple hours, I would like to do that.    

I also need to understand your intent about C#-7.    Normally it is pretty easy to avoid new features for a while and that is what we do.   That would allow this PR to proceed more easily.  

If you intend to immediately use C#-7 features, or if supporting both is hard then we need to warn people in the readme/markdown, and create a commit tag for VS2015 users and add guidance in markdown for using that tag.